### DISCUSSION TITLE
Icon genie not working

### DISCUSSION THREAD

**Discussion by ornellas13** [quasarframework/quasar#11995] [Quasar App Extensions] [answered]
I do all the process installing icon genie, by cli generate all icons and splashscreen, but when just type "icongenie verify" on the Mode CORDOVA in all files shows this message after the path file: 

Mode CORDOVA 

 SIZE OK - png:          src-cordova/res/android/ldpi.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/android/mdpi.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/android/hdpi.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/android/xhdpi.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/android/xxhdpi.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/android/xxxhdpi.png ERROR: platform not installed!
 SIZE OK - splashscreen: src-cordova/res/screen/android/splash-land-ldpi.png ERROR: platform not installed!
 SIZE OK - splashscreen: src-cordova/res/screen/android/splash-port-ldpi.png ERROR: platform not installed!
 SIZE OK - splashscreen: src-cordova/res/screen/android/splash-land-mdpi.png ERROR: platform not installed!
 SIZE OK - splashscreen: src-cordova/res/screen/android/splash-port-mdpi.png ERROR: platform not installed!
 SIZE OK - splashscreen: src-cordova/res/screen/android/splash-land-hdpi.png ERROR: platform not installed!
 SIZE OK - splashscreen: src-cordova/res/screen/android/splash-port-hdpi.png ERROR: platform not installed!
 SIZE OK - splashscreen: src-cordova/res/screen/android/splash-land-xhdpi.png ERROR: platform not installed!
 SIZE OK - splashscreen: src-cordova/res/screen/android/splash-port-xhdpi.png ERROR: platform not installed!
 SIZE OK - splashscreen: src-cordova/res/screen/android/splash-land-xxhdpi.png ERROR: platform not installed!
 SIZE OK - splashscreen: src-cordova/res/screen/android/splash-port-xxhdpi.png ERROR: platform not installed!
 SIZE OK - splashscreen: src-cordova/res/screen/android/splash-land-xxxhdpi.png ERROR: platform not installed!
 SIZE OK - splashscreen: src-cordova/res/screen/android/splash-port-xxxhdpi.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon@2x.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-20@2x.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-20@3x.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-29.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-29@2x.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-29@3x.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-40@2x.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-60@2x.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-60@3x.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-20.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-20@2x.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-40.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-50.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-50@2x.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-72.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-72@2x.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-76.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-76@2x.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-83.5@2x.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-1024.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-24@2x.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-27.5@2x.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-29@2x.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-29@3x.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-40@2x.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-44@2x.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-50@2x.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-86@2x.png ERROR: platform not installed!
 SIZE OK - png:          src-cordova/res/ios/icon-98@2x.png ERROR: platform not installed!
 SIZE OK - splashscreen: src-cordova/res/screen/ios/Default@2x~iphone~anyany.png ERROR: platform not installed!
 SIZE OK - splashscreen: src-cordova/res/screen/ios/Default@2x~iphone~comany.png ERROR: platform not installed!
 SIZE OK - splashscreen: src-cordova/res/screen/ios/Default@2x~iphone~comcom.png ERROR: platform not installed!
 SIZE OK - splashscreen: src-cordova/res/screen/ios/Default@3x~iphone~anyany.png ERROR: platform not installed!
 SIZE OK - splashscreen: src-cordova/res/screen/ios/Default@3x~iphone~anycom.png ERROR: platform not installed!
 SIZE OK - splashscreen: src-cordova/res/screen/ios/Default@3x~iphone~comany.png ERROR: platform not installed!
 SIZE OK - splashscreen: src-cordova/res/screen/ios/Default@2x~ipad~anyany.png ERROR: platform not installed!
 SIZE OK - splashscreen: src-cordova/res/screen/ios/Default@2x~ipad~comany.png ERROR: platform not installed!

Any guess?
Thanks

**Comment by MilosPaunovic** [comment]
Have you added the platforms?
[https://quasar.dev/quasar-cli/developing-cordova-apps/preparation#3-add-platform](https://quasar.dev/quasar-cli/developing-cordova-apps/preparation#3-add-platform)

**Comment by ornellas13** [comment]
Sure, yes.

First of all, thank you so much to answer my question, I'm so glad, thank you.

So, I was already running this project correctly when had a moment that Google Play ask me to upgrade to level 29 for publish at the store.

So, again, I did what I had to do to work correctly.

But, at that moment when I did run "quasar dev -m android", a trouble appear with the Icon Genie. Until this moment, I was running the version 1 of Icon Genie. So, I did uninstall it and follow the instructions at the Icon Genie page, running "npm install -g @quasar/icongenie" and after "quasar ext remove @quasar/icon-genie".

The quasar project did come back to work fine, when I run "quasar build -m android" generates a ".aab" file and apply it to the store. However, when I did upgrade the app, the big deal is the icons and splashscreen was gone.

So, I did back to my quasar project, run again:
-> icongenie generate -i /path/to/icon.png
Also:
-> icongenie g -m cordova -i /path/to/icon.png

And it "creates correctly". The archives is inside the folders correctly. But, when I run:
-> icongenie verify

Shows those errors mentioned in the first post.

When I run "quasar build -m android" the real time log doesn't show anymore that message:

"App: Running "@quasar/icon-genie" Quasar App Extension..."

I don't know if should, whatever.
I really did a lot of things, like try to delete and recreate the folder "src-cordova", try to use the files "package.json" and "package-lock.json" of my qemu apple virtual machine, used to bluid the iOs version, but nothing works.

I really thank you all that can help me to find a solution.

Peace!

**Comment by ornellas13** [comment]
Any help?

**Comment by ornellas13** [comment, marked as answer]
I found the solution. I compared my config.xml and realize that the attribute name="android" inside platform tag had disappear.
